.gitignore

```diff
@@ -1,2 +1,2 @@
 /node_modules/
-.DS_Store
+.DS_Store
```

Comment:
🤔 I thought I added this here: https://github.com/mdn/data/commit/4f74b576de422743b13752064355857610c3b4fc not sure what the difference is

Reply:
I added it too, in this PR, so it ended up creating a merge conflict. That's why it looks a little odd.

Reply:
Gotcha